.evergreen/run-tests.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       AUTH                    Set to enable authentication. Defaults to "noauth"
+#       SSL                     Set to enable SSL. Defaults to "nossl"
+#       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
+#       MARCH                   Machine Architecture. Defaults to lowercase uname -m
+
+
+AUTH=${AUTH:-noauth}
+SSL=${SSL:-nossl}
+MONGODB_URI=${MONGODB_URI:-}
+TESTS=${TESTS:-}
+IS_MATRIX_TESTING=${IS_MATRIX_TESTING:-false}
+
+# For matrix testing, we have to determine the correct driver version
+if [ "$IS_MATRIX_TESTING" == "true" ]; then
+   case "${DRIVER_MONGODB_VERSION}" in
+      '4.4')
+         export EXTENSION_VERSION='1.8.2'
+         ;;
+      '4.2')
+         export EXTENSION_VERSION='1.6.1'
+         ;;
+      '4.0')
+         export EXTENSION_VERSION='1.5.5'
+         ;;
+   esac
+
+   case "${MONGODB_VERSION}" in
+      latest)
+         MONGODB_VERSION_NUMBER='5.0'
+         ;;
+      *)
+         MONGODB_VERSION_NUMBER=$MONGODB_VERSION
+         ;;
+   esac
+
+   PHPUNIT_OPTS="--dont-report-useless-tests --exclude-group matrix-testing-server-${MONGODB_VERSION_NUMBER}-driver-${DRIVER_MONGODB_VERSION},matrix-testing-server-${MONGODB_VERSION_NUMBER}-driver-${DRIVER_MONGODB_VERSION}-topology-${TOPOLOGY}"
```

Comment:
Very nice.

Reply:
Yeah, not happy about the formatting, but your suggestion to use groups to skip tests was great.